yarn.lock

```diff
@@ -2584,14 +2591,26 @@ charenc@0.0.2:
   resolved "https://registry.yarnpkg.com/charenc/-/charenc-0.0.2.tgz#c0a1d2f3a7092e03774bfa83f14c0fc5790a8667"
   integrity sha512-yrLQ/yVUFXkzg7EDQsPieE/53+0RlaWTs+wBrvW36cyilJ2SaDWfl4Yj7MtLTXleV9uEKefbAGUPv2/iWSooRA==
 
-chart.js@^2.6.0, chart.js@^2.9.4:
+chart.js@^2.6.0:
```

Comment:
We shouldn't have anything still using the old version - what is causing this to be kept around?